File: OdontoSysFrontEnd/OdontoSysModel/Infraestructure/Turno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OdontoSysModel.Infraestructure
{
    public class Turno
    {
        public int IdTurno { get; set; }
        public DateTime HoraInicio { get; set; }
        public DateTime HoraFin { get; set; }
        public DiaSemana DiaSemana { get; set; }

        public Turno()
        {
        }

        public override string ToString()
        {
            return $"Turno{{IdTurno={IdTurno}, HoraInicio={HoraInicio}, HoraFin={HoraFin}, DiaSemana={DiaSemana}}}";
        }
    }
}

File: BackEnd/OdontoSysModel/src/main/java/pe/pucp/edu/odontosys/infrastructure/model/Turno.java
package pe.pucp.edu.odontosys.infrastructure.model;

public class Turno {
    
    private Integer idTurno;
    
    private String horaInicio;
    
    private String horaFin;
    
    private DiaSemana diaSemana;

    public Turno() {
    }

    public Integer getIdTurno() {
        return idTurno;
    }

    public void setIdTurno(Integer idTurno) {
        this.idTurno = idTurno;
    }

    public String getHoraInicio() {
        return horaInicio;
    }

    public void setHoraInicio(String horaInicio) {
        this.horaInicio = horaInicio;
    }

    public String getHoraFin() {
        return horaFin;
    }

    public void setHoraFin(String horaFin) {
        this.horaFin = horaFin;
    }

    public DiaSemana getDiaSemana() {
        return diaSemana;
    }

    public void setDiaSemana(DiaSemana diaSemana) {
        this.diaSemana = diaSemana;
    }

    @Override
    public String toString() {
        String resultado = "";
        resultado += "================\n";
        resultado += "Turno\n";
        resultado += "----------------\n";
        resultado += "idTurno: " + idTurno + "\n";
        resultado += "horaInicio: " + horaInicio + "\n";
        resultado += "horaFin: " + horaFin + "\n";
        resultado += "diaSemana: " + diaSemana + "\n";
        resultado += "================\n";
        return resultado;
    }
     
}
